.pipelines/templates/template-az-aro-extension-prod.yml

```diff
@@ -0,0 +1,5 @@
+steps:
```

Comment:
why is the prod extension needed? either use the latest upstream az as-is or the plugin in this repo

Reply:
I was actually using the latest released from prod in a docker container, but you requested to change it to use the internal here https://github.com/Azure/ARO-RP/pull/643#discussion_r421034686 should I change it back to latest release via docker? Or do `make pyenv az` ?